lib/evss/error_middleware.rb

```diff
@@ -7,7 +7,7 @@ def on_complete(env)
       case env[:status]
       when 200
         resp = env.body
-        raise EVSSError, resp['messages'] if resp['success'] && !resp['success']
+        raise EVSSError, resp['messages'] if 'success'.in?(resp) && !resp['success']
```

Comment:
It might be clearer to just explicitly check the value, I think I originally only had the negation check but then saw the error being raised for responses that didn't include the success flag so I quickly tacked on the additional check
 `if resp['success'] == false` should work and only be true when the success flag is present and false